packages/cli/test/composites.test.ts

```diff
@@ -24,9 +36,177 @@ describe('composites', () => {
         'test/mocks/composite.schema',
         `--key=${seed}`,
       ])
-      expect(create.stderr.toString().includes('"version": "1.0",')).toBe(true)
-      expect(create.stderr.toString().includes('"aliases":')).toBe(true)
-      expect(create.stderr.toString().includes('"views":')).toBe(true)
+      expect(create.stdout.toString().includes('"version": "1.0",')).toBe(true)
+      expect(create.stdout.toString().includes('"aliases":')).toBe(true)
+      expect(create.stdout.toString().includes('"views":')).toBe(true)
+    }, 60000)
+  })
+
+  describe('composite:deploy', () => {
+    let wasModelLoaded: boolean
+    const waitForModelToLoad = async (
+      ceramic: CeramicClient,
+      modelStreamID: string,
+      timeout: number
+    ) => {
+      await Promise.race([
+        new Promise((resolve) =>
+          setTimeout(() => {
+            resolve(true)
+          }, timeout)
+        ),
+        Model.load(ceramic, modelStreamID).then((model) => {
+          wasModelLoaded = true
+          return model
+        }),
+      ])
+    }
```

Comment:
What is this logic for please? Seems it should be test-specific, not having a mutable variable between tests.
Also, shouldn't the timeout reject the promise rather than resolve?

Reply:
I updated this function to make more sense. Please, have a look :) .